code/modules/projectiles/magazines/specialist.dm

```diff
@@ -252,6 +252,52 @@
 /obj/item/ammo_magazine/internal/launcher/rocket/m57a4/ds
 	default_ammo = /datum/ammo/rocket/wp/quad/ds
 
+//SOM RPG
+/obj/item/ammo_magazine/rocket/som
+	name = "\improper 84mm light-explosive RPG"
+	desc = "A high explosive warhead for the V-71 rocket launcher. Causes a strong explosion over a respectable area."
+	icon_state = "rpg_rocket_le"
+	default_ammo = /datum/ammo/rocket/som
+	reload_delay = 2 SECONDS
+	bonus_overlay = "rpg_rocket"
+
+/obj/item/ammo_magazine/rocket/som/light
+	name = "\improper 84mm light-explosive RPG"
+	desc = "A light explosive warhead for the V-71 rocket launcher. Causes a light explosion over a large area but low impact damage."
+	icon_state = "rpg_rocket_le"
+	default_ammo = /datum/ammo/rocket/som/light
+	reload_delay = 1 SECONDS
+	bonus_overlay = "rpg_rocket_le"
+
+/obj/item/ammo_magazine/rocket/som/heat
+	name = "\improper 84mm HEAT RPG"
+	desc = "A high explosive anti armor warhead for the V-71 rocket launcher. Designed to punch through the toughest armor."
+	icon_state = "rpg_rocket_heat"
+	default_ammo = /datum/ammo/rocket/som/heat
+	bonus_overlay = "rpg_rocket_heat"
+
+/obj/item/ammo_magazine/rocket/som/thermobaric
+	name = "\improper 84mm thermobaric RPG"
+	desc = "A thermobaric warhead for the V-71 rocket launcher. Causes a powerful fuel air explosion over a moderate area."
+	icon_state = "rpg_rocket_thermobaric"
+	default_ammo = /datum/ammo/rocket/som/thermobaric
+	bonus_overlay = "rpg_rocket_thermobaric"
+
+/obj/item/ammo_magazine/rocket/som/rad
+	name = "\improper 84mm  irrad RPG"
```

Comment:
```suggestion
	name = "\improper 84mm tactical nuclear RPG"
```
"Radiological" works too, but it is up to you

Reply:
Calling it nuclear implies it's actually causing a nuclear reaction, this is more of a dirty bomb, it's just a conventional explosive that's also irradiating people.